manager/src/main/java/com/redhat/service/bridge/manager/connectors/ConnectorsOrchestratorImpl.java

```diff
@@ -54,6 +55,8 @@
             return eventBus.request(CONNECTOR_CREATED_EVENT, c);
         } else if (c.getDesiredStatus().equals(ConnectorStatus.READY) && c.getStatus() == ConnectorStatus.TOPIC_CREATED) {
             return eventBus.request(KAFKA_TOPIC_CREATED_EVENT, c);
+        } else if (c.getDesiredStatus().equals(ConnectorStatus.READY) && c.getStatus() == ConnectorStatus.MANAGED_CONNECTOR_CREATED) {
```

Comment:
I added support to the orchestration mechanism too.. ideally all of these events should have re-try configuration?

Reply:
Currently it's disable, so you can either update it or not

Reply:
I've now enabled it.